privacyidea/static/components/config/views/config.mresolvers.ldap.html

```diff
@@ -94,6 +94,21 @@
                    placeholder="500"/>
         </div>
     </div>
```

Comment:
You can remove the complete change in this file, since the machine resolver is not editable.
We do not support adding machine objects to ldap :-)